mappings/1.8.2-pre4#1.8.2-pre5.tinydiff

```diff
@@ -108,10 +108,10 @@ c	net/minecraft/C_13piauvdk
 		p	10			p_10
 	m	()Ljava/lang/String;	m_70qzhhnku		getCurrentAction
 c	net/minecraft/C_25obpltwe
-	m	(FFF)I	m_11fmbeynf		m_11fmbeynf
-		p	0			p_0
-		p	1			p_1
-		p	2			p_2
+	m	(FFF)I	m_11fmbeynf		toRgb
+		p	0			hue
+		p	1			saturation
+		p	2			value
```

Comment:
Assuming this converts HSB to RGB, this should be brightness?

Reply:
It's HSV

Reply:
Apparently they're the same thing? In any case this is good to go then.